scripts/netContain/Dockerfile

```diff
@@ -13,19 +13,18 @@
 #limitations under the License.
 ##
 
-# One Container for OVS / netplugin / netmaster 
+# One Container for OVS / netplugin / netmaster
 
-FROM ubuntu:16.04
+FROM alpine:3.5
```

Comment:
Any reason for not using the latest 3.6 version?

Reply:
@neelimamukiri: I was testing with 3.5 as with v2plugin. I need to check that it works properly.